### PR TITLE
chore: add internal page to command menu

### PR DESCRIPTION
**Pull request by sarahgm**
# Description

add internal page to command menu

# Reviewers:

@marigold-ui/developer
@marigold-ui/designer

### CODE REVIEW

File: docs/app/_components/CommandItems.tsx [hunk 1]
```diff
@@ -255,3 +255,33 @@ export const PagesItem = ({
     </CommandGroup>
   );
 };
+
+export const InternalPage = ({ classNames, setOpen }: ChangeOpenItemProps) => {
+  const router = useRouter();
+  const params = useSearchParams();
+
+  const goto = ({ slug, hash = '' }: { slug: string; hash?: string }) => {
```
Comment (aromko) [inline docs/app/_components/CommandItems.tsx:263]: Can't we reuse the goto function from PagesItem? Only `setPages([]);` is missing. Don't we need this?

Comment (sarahgm) [reply docs/app/_components/CommandItems.tsx:263]: we don't need setPages because its for the subitems

Comment (sarahgm) [reply docs/app/_components/CommandItems.tsx:263]: but I will have a look again and see if I can reuse it :)

Comment (sarahgm) [reply docs/app/_components/CommandItems.tsx]: can you have a look again please? :) @aromko

**Review by aromko — APPROVED** [review]

**Pull request merged**